qiskit/transpiler/preset_passmanagers/level1.py

```diff
@@ -169,7 +169,9 @@ def _swap_condition(property_set):
             Unroll3qOrMore(),
             Collect2qBlocks(),
             ConsolidateBlocks(basis_gates=basis_gates),
-            UnitarySynthesis(basis_gates, approximation_degree=approximation_degree),
+            UnitarySynthesis(
+                basis_gates, approximation_degree=approximation_degree, coupling_map=coupling_map
```

Comment:
I think the `backend_props` should also be passed in these preset passmanagers so that when compiling for a device with symmetric coupling_map but non-symmetric durations the pulse-efficient decomposition is done.

Reply:
(plus a test for this scenario)

Reply:
I agree we should be setting the properties in the preset pass managers. But reading the code above, I'm not sure this actually works as you describe it. The condition logic is if natural direction detection from unitary synthesis is:
```
if self._natural_direction in {None, True} and layout and self._coupling_map:
    ...
elif self._natural_direction in {None, True} and layout and self._backend_props:
    ...
```
So even if the coupling map is non-symmetric I don't think it goes and ever checks the backend properties because a coupling map is set.

Reply:
I'll check if they exist

Reply:
Ok, this should be fixed now with: https://github.com/Qiskit/qiskit-terra/pull/6581/commits/0c67c81a91d360850a67f9c771dfc46769199fed